how-to/register-with-browser/client/src/browser.ts

```diff
@@ -0,0 +1,39 @@
+import { LayoutExtended } from '@openfin/workspace';
+import { getCurrentSync, PageLayout, PageWithUpdatableRuntimeAttribs, WorkspacePlatformModule } from '@openfin/workspace-platform';
+import { v4 } from 'uuid';
+
+const platform: WorkspacePlatformModule = getCurrentSync();
+
+export async function launchView(view:OpenFin.PlatformViewCreationOptions | string , targetIdentity?: OpenFin.Identity){
+    let viewOptions;
+    if(typeof view === "string"){
+        viewOptions = { "url": view};
+    } else {
+        viewOptions = view;
+    }
+    return platform.createView(viewOptions, targetIdentity);
+}
+
+export async function createPageLayout(layout): Promise<PageLayout> {
+    const layoutId: string = `layout-${v4()}`;
+    return {
+        ...layout,
+        layoutDetails: { layoutId }
+    } as PageLayout;
+}
+
+export async function createPageWithLayout(title: string, layout: LayoutExtended): Promise<PageWithUpdatableRuntimeAttribs> {
+    const layoutWithDetails = await createPageLayout(layout);
+    return {
+        pageId: v4(),
```

Comment:
We don't need to import uuid.

You can use: let uuid = window.crypto.randomUUID();

Reply:
this throws a type error, going to use date.now for now 😄

Reply:
:) yeah I think the version of typescript sometimes doesn't recognise that and it has to be window.crypto["randomUUID"]() or it gets cast.

Reply:
all set i think 😄